pages/services/marathon-lb/advanced/index.md

```diff
@@ -31,48 +31,86 @@ Marathon-LB has a templating feature for specifying custom HAProxy configuration
+      ssl-default-bind-options no-sslv3 no-tlsv10 no-tls-tickets
+      ssl-default-server-ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:DHE-RSA-AES128-SHA256:DHE-RSA-AES256-SHA256:AES128-GCM-SHA256:AES256-GCM-SHA384:AES128-SHA256:AES256-SHA256:!aNULL:!MD5:!DSS
+      ssl-default-server-options no-sslv3 no-tlsv10 no-tls-tickets
+      stats socket /var/run/haproxy/socket expose-fd listeners
+      server-state-file global
+      server-state-base /var/state/haproxy/
+      lua-load /marathon-lb/getpids.lua
+      lua-load /marathon-lb/getconfig.lua
+      lua-load /marathon-lb/getmaps.lua
+      lua-load /marathon-lb/signalmlb.lua
+    defaults
+      load-server-state-from-file global
+      log               global
+      retries                   3
+      backlog               10000
+      maxconn                3000
+      timeout connect          5s
+      timeout client          30s
+      timeout server          30s
+      timeout tunnel        3600s
+      timeout http-keep-alive  1s
+      timeout http-request    15s
+      timeout queue           30s
+      timeout tarpit          60s
+      option            dontlognull
+      option            http-server-close
+      option            redispatch
+    listen stats
+      bind 0.0.0.0:9090
+      balance
+      mode http
+      stats enable
+      monitor-uri /_haproxy_health_check
+      acl getpid path /_haproxy_getpids
+      http-request use-service lua.getpids if getpid
+      acl getvhostmap path /_haproxy_getvhostmap
+      http-request use-service lua.getvhostmap if getvhostmap
+      acl getappmap path /_haproxy_getappmap
+      http-request use-service lua.getappmap if getappmap
+      acl getconfig path /_haproxy_getconfig
+      http-request use-service lua.getconfig if getconfig
+
+      acl signalmlbhup path /_mlb_signal/hup
+      http-request use-service lua.signalmlbhup if signalmlbhup
+      acl signalmlbusr1 path /_mlb_signal/usr1
+      http-request use-service lua.signalmlbusr1 if signalmlbusr1
+    ```
+
+    In the code above, the following items have changed from the default: `maxconn`.
+
+    The current `HAPROXY_HEAD`, as well as other Marathon templates, can be [found here](https://github.com/mesosphere/marathon-lb/blob/master/Longhelp.md#haproxy_head").
+
+2.  Tar or zip the file. 
+
+    ```bash
+    tar czf templates.tgz templates/
+    ```
+
+    Take the file you created (`templates.tgz` if you use the script), and make it available from an HTTP server. 
+
+3.  Augment the Marathon-LB config by copying the following JSON into a file called `options.json`, putting in the URL to your file:
```

Comment:
So I looked at the step below that one.... and saw that it points the template-url to that old download again. So I am editing that line too, to instruct the user to put the URL of their own file in the options.json

Reply:
Yes, this looks good.  My only other comment is on line 107, we should change:
(`templates.tgz` if you use the script)   --> (`templates.tgz`)
Because, we got rid of the 'the script'.

Reply:
done :)